docs/docs/tutorials/using_dask.md

```diff
@@ -57,9 +61,9 @@ options = gateway.cluster_options()
 options
 ```
 
-![Nebari - Cluster Options UI](/img/tutorials/cluster_options.png)
+![Nebari - Cluster Options UI](/img/tutorials/dask_cluster_options.png)
```

Comment:
This screenshot needs to be updated pending the question on the name of the default dask env.

Reply:
Done!